### ISSUE TITLE
Bronchial wall thickening

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6573]
**Preferred term label:**  Bronchial wall thickening

**Synonyms**

**Definition (free text, please give PubMed ID)**
Abnormal thickening of the bronchial walls

**Parent term (use hpo.jax.org/app)**
Abnormal bronchus morphology HP:0025426

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
@eswietlik Is this the sign you are referring to and should we add these synonyms?
https://en.wikipedia.org/wiki/Peribronchial_cuffing

**Comment by pnrobinson** [comment]
added with this definition

Radiological appearance of increased density around the walls of a bronchus or large bronchiole. This feature is thought to be related to edema involving the bronchial wall as well as the peribronchial interstitial space. If the cross section of a bronchus is captured in a radiograph or computed tomography image, it is said to have the appearance of a donut because of the central lucency representing the airway of the bronchus surrounded by a circular region of increased density.
